profile/README.md

```diff
@@ -1,3 +1,30 @@
 # California Cropland Monitoring and Modeling Framework
 
 ## About
+
+The California Cropland Monitoring and Modeling Framework (CCMMF) will be a biogeochemical model and data integration pipeline to generate inventories and future projections of soil carbon (SOC) stocks and greenhouse gas fluxes (GHG) from California’s croplands. 
+CCMMF will leverage Bayesian techniques to combine heterogeneous datasets into unified wall-to-wall inventories and climate and scenario-based projections. 
+Data layers will be generated as ensemble projections to facilitate propagation of uncertainty through downstream applications.
+
+CCMMF will produce new gridded time series of key agronomic practices including planting, harvest, irrigation, and tillage derived from satellite 
+measurements and agronomic statistics. 
+The biogeochemistry model will be built on SIPNET, expanded to represent agronomic management, nitrogen cycling, and fluxes of nitrous oxide and methane. 
+Using remotely sensed practices to drive the model will produce consistent model estimates that do not depend on obtaining records from individual farmers. 
```

Comment:
"We will utilize existing and newly-derived remote sensing of annual land management and agricultural practices ..."

Reply:
```suggestion
We will utilize existing and newly-derived remote sensing of annual land management and agricultural practices to drive the model will produce consistent model estimates that do not depend on obtaining records from individual farmers. 
```